pontoon/base/templates/translate.html

```diff
@@ -239,27 +239,34 @@ <h3 class="no-match"><div class="fa fa-exclamation-circle"></div>No results</h3>
         <li><a href="#plural6"><span></span><sup></sup></a></li>
       </ul>
     </nav>
-    <textarea dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
-
+    {% if user.is_authenticated() %}
+      <textarea dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
+    {% else %}
+      <textarea readonly dir="auto" lang="{{ locale.code }}" id="translation" placeholder="Type translation and press Enter to save"></textarea>
+    {% endif %}
     <menu>
-      <div id="unsaved" class="warning-overlay">
-        <a href="#" class="cancel">&times;</a>
-        <h2>You have unsaved changes</h2>
-        <p>Sure you want to leave?</p>
-        <button id="leave-anyway">Leave anyway</button>
-      </div>
-      <div id="quality" class="warning-overlay">
-        <a href="#" class="cancel">&times;</a>
-        <h2>The following checks have failed</h2>
-        <ul></ul>
-        <button id="save-anyway"></button>
-      </div>
-      <div id="translation-length">
-        <span class="current-length"></span>|<span class="original-length"></span>
-      </div>
-      <button id="copy" title="Copy From Source (Alt + C)">Copy</button>
-      <button id="clear" title="Clear Translation (Alt + Backspace)">Clear</button>
-      <button id="save" title="Save Translation (Enter)"></button>
+      {% if user.is_authenticated() %}
+        <div id="unsaved" class="warning-overlay">
+          <a href="#" class="cancel">&times;</a>
+          <h2>You have unsaved changes</h2>
+          <p>Sure you want to leave?</p>
+          <button id="leave-anyway">Leave anyway</button>
+        </div>
+        <div id="quality" class="warning-overlay">
+          <a href="#" class="cancel">&times;</a>
+          <h2>The following checks have failed</h2>
+          <ul></ul>
+          <button id="save-anyway"></button>
+        </div>
+        <div id="translation-length">
+          <span class="current-length"></span>|<span class="original-length"></span>
+        </div>
+        <button id="copy" title="Copy From Source (Alt + C)">Copy</button>
+        <button id="clear" title="Clear Translation (Alt + Backspace)">Clear</button>
+        <button id="save" title="Save Translation (Enter)"></button>
+      {% else %}
+        <p><i>You need to be logged in to suggest or make changes.</i></p>
```

Comment:
You should use CSS for styling instead of markup.

Another nit: please you something like "Sign in to suggest or make changes.", because we use sign in instead of log in elsewhere and because it's shorter and as such less likely to break into two lines.